src/main/resources/edu/cornell/kfs/ksr/cu-ojb-ksr.xml

```diff
@@ -0,0 +1,30 @@
+<descriptor-repository version="1.0">
+
+	<class-descriptor class="edu.cornell.kfs.ksr.businessobject.SecurityGroup" table="KRSR_SEC_GRP_T">
+		<field-descriptor name="securityGroupId" column="SEC_GRP_ID" jdbc-type="BIGINT" primarykey="true" sequence-name="KRSR_SEC_GRP_ID_SEQ" autoincrement="true" />
+		<field-descriptor name="securityGroupName" column="SEC_GRP_NM" jdbc-type="VARCHAR" />
+		<field-descriptor name="securityGroupDescription" column="SEC_GRP_DESC" jdbc-type="VARCHAR" />
+		<field-descriptor name="active" column="actv_ind" jdbc-type="VARCHAR" conversion="org.kuali.kfs.core.framework.persistence.ojb.conversion.OjbCharBooleanConversion" />
+		<field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" />
+		<field-descriptor name="versionNumber" locking="true" column="VER_NBR" jdbc-type="BIGINT" />
+		<field-descriptor name="lastUpdatedTimestamp" column="LAST_UPDT_TS" jdbc-type="TIMESTAMP" index="true"/>
+		
+	    <collection-descriptor name="securityGroupTabs" element-class-ref="edu.cornell.kfs.ksr.businessobject.SecurityGroupTab" collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList"
```

Comment:
I'm seeing a mix of tabs and spaces in this file. Can it be easily simplified to use just spaces?

Reply:
updated